### PR TITLE
update Readme

### PR DESCRIPTION
**Pull request by boneyard93501**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,12 +16,15 @@
 
 ## Table of Contents
 
-- [Install](#install)
-- [Usage](#usage)
-- [Supported Hash Types](#supported-hash-types)
-- [Maintainers](#maintainers)
-- [Contribute](#contribute)
-- [License](#license)
+- [rust-multihash](#rust-multihash)
```
Comment (vmx) [inline README.md:19]: I think the README is small enough to keep the current table of contents. What do you think @boneyard93501?

**Review by vmx — CHANGES_REQUESTED** [review]
Thanks for updating, I'd prefer if only the code would be changed and the current table of contents would be kept.

File: README.md (hunk 1 above)
Comment (boneyard93501) [reply README.md:19]: absolutely. for some reason, vscode likes to rearrange the list and i didn't notice.
```
- [Table of Contents](#table-of-contents)
  - [Install](#install)
  - [Usage](#usage)
      - [rust-multihash](#rust-multihash)
      - [Using a custom code table](#using-a-custom-code-table)
  - [Supported Hash Types](#supported-hash-types)
  - [Maintainers](#maintainers)
  - [Contribute](#contribute)
  - [License](#license)
``` 
work>

**Review by vmx — APPROVED** [review]
The table of contents now also lists the table of contents itself. Though this is really a minor thing. If you feel like removing that, it would be great, but if you cannot be bothered, I'll merge it within the next days as it currently is.

**Comment by vmx** [comment]
Thank you @boneyard93501, I totally forgot to merge this one.

**Pull request merged**